static/panes/editor.js

```diff
@@ -1428,11 +1431,33 @@ Editor.prototype.onCompileResponse = function (compilerId, compiler, result) {
                     endColumn: colEnd,
                     code: link,
                 };
+                if (obj.tag.fixes) {
+                    fixes.push(
+                        ...obj.tag.fixes.map((fs, ind) => {
```

Comment:
For JS code we're still stuck on es5, so we need to take care not to introduce es6 features here. Note that we're working on moving everything over to TS so this won't be a restriction for much longer, but until then, could you downgrade this? Sorrya about that

Reply:
I'm not very knowledgeable of what is ES5 or ES6. Is it no longer using any ES6 feature now?